sql/index.go

```diff
@@ -479,15 +482,17 @@ func (r *IndexRegistry) AddIndex(idx Index) (chan<- struct{}, error) {
 	r.indexOrder = append(r.indexOrder, key)
 	r.mut.Unlock()
 
-	var created = make(chan struct{})
+	var _created = make(chan struct{})
```

Comment:
Just curious about the underscore prefix, Is it frequently used or kind of standard in the community?

Reply:
Nope, `created` and `ready` were used as named return arguments in the signature for documentation purposes, but they are restricted to one direction, so I can't really use them inside